skopt/acquisition.py

```diff
@@ -150,7 +150,7 @@ def gaussian_pi(X, model, y_opt=0.0, xi=0.01, return_grad=False):
     values[mask] = norm.cdf(scaled)
 
     if return_grad:
-        if not np.any(mask):
+        if not np.all(mask):
```

Comment:
So if mask has more than one value, we don't support it. It raises an error before this line is reached when `predict` is called.

Anyway I can make that clearer?